source/Halibut.Tests/Util/SerilogLoggerBuilder.cs

```diff
@@ -1,19 +1,68 @@
+using System;
+using System.IO;
+using NUnit.Framework;
 using Serilog;
+using Serilog.Core;
+using Serilog.Events;
 using Serilog.Formatting.Display;
 using Serilog.Sinks.NUnit;
 
 namespace Halibut.Tests.Util
 {
     public class SerilogLoggerBuilder
     {
-        const string OutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
+        const string OutputTemplate = "{Timestamp:HH:mm:ss.fff zzz} {Message}{NewLine}{Exception}";
         
         public ILogger Build()
         {
+            // In teamcity we need to know what test the log is for, since we can find hung builds and only have a single file containing all log messages.
```

Comment:
I don't mind this being a TeamCity specific thing.

But I was wondering, why not just have it this way in general? Maybe it'll help the local experience as well?

Just a thought.

Reply:
progress is nicer locally since that will show logs in your ide immediately, otherwise you have to wait until the test completes to see what happened. This is very problematic when a mistake is made and the test never finishes :D